main.nf

```diff
@@ -4,7 +4,7 @@
     nf-core/epitopeprediction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Github : https://github.com/nf-core/epitopeprediction
-    Website: https://nf-co.re/epitopeprediction
+Website: https://nf-co.re/epitopeprediction
```

Comment:
Is that intended? Now it seems to be not aligned any more... 
edit: Well, its also in my ampliseq PR so seems to be intended.